django/contrib/admin/static/admin/js/popup_response.js

```diff
@@ -3,6 +3,9 @@
     'use strict';
     var initData = JSON.parse(document.getElementById('django-admin-popup-response-constants').dataset.popupResponse);
     switch(initData.action) {
+    case 'add':
```

Comment:
The same hack that this ticket is based on.
The popup will not have the "Yes, I'm sure" button.